src/interfaces/stepper-interfaces/CreateShareStructure/create-share-structure-interface.ts

```diff
@@ -0,0 +1,18 @@
+export interface ShareStructureIF {
+    valid: boolean;
+    shareClasses : ShareClassIF[];
+}
+
+export interface ShareClassIF {
+    id: number| null;
+    displayOrder: number; // To support move up and down functionality
```

Comment:
Or do we want to simply reorder the array of shares?

Reply:
Then again, when we restore a draft, we can't guarantee the order of an array, so leave this in.